packages/vuetify/src/components/VField/VField.sass

```diff
@@ -56,7 +56,11 @@
       @include tools.density('v-input', $input-density) using ($modifier)
         @at-root #{selector.nest(&, $root)}
           --v-input-control-height: #{$field-control-height + $modifier}
-          --v-field-padding-bottom: #{$field-control-padding-bottom + $modifier * .5}
+          $v-field-padding-bottom: ($field-control-padding-bottom + $modifier * .5)
```

Comment:
this padding-bottom can be a negative value, however, browser renders negative padding as 0.

So this if else logic ensure to return 0 when calculate outcome is negative, so it won't impact min-height calculation